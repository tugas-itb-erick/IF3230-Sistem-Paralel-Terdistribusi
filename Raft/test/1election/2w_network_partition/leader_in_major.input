5
set_timeout 5 0
simulate 10
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
disconnect 1 4
disconnect 1 5
disconnect 2 4
disconnect 2 5
disconnect 3 4
disconnect 3 5
set_timeout 1 0
simulate 10
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
simulate 100
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
connect 1 4
connect 1 5
connect 2 4
connect 2 5
connect 3 4
connect 3 5
set_timeout 2 0
timestep
flush_outbox
timestep
flush_outbox
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
end_simulation