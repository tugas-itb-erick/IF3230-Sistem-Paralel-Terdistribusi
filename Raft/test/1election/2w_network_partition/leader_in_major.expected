follower
follower
follower
follower
leader
leader
follower
follower
follower
leader
leader
follower
follower
follower
leader
follower
leader
follower
follower
follower
